samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1057,71 +1062,49 @@ The following sections outline the various onboarding methods that this sample s
 
 .. _nrf_cloud_multi_service_install_nrf_utils:
 
-Download and install nRF Cloud Utilities
+Installing nRF Cloud Utils
 ========================================
 
-To perform many of the actions in the other sections, you will need to install the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repository.
+To perform many of the actions described in the other sections, you need to install the `nRF Cloud Utils <nRF Cloud Utils_>`_.
 This is not necessary if you are using the `auto-onboarding <nRF Cloud Auto-onboarding_>`_ feature of the nRF Cloud Provisioning Service.
 
-To install the repository, clone or download it and `install the required packages <nRF Cloud Utilities Prerequisites_>`_.
+To install the nRF Cloud Utils, run the following command to use this package as a dependency:
+
+.. parsed-literal::
+   :class: highlight
+
+   pip3 install nrfcloud-utils
 
 .. _nrf_cloud_multi_service_provisioning_service:
 
 Provisioning with the nRF Cloud Provisioning Service
 ====================================================
 
 The nRF Cloud Provisioning Service is only compatible with nRF91x1 devices.
-For devices using the nRF9160, proceed to :ref:`nrf_cloud_multi_service_standard_onboarding`.
-
-If you have :ref:`enabled support <nrf_cloud_multi_service_building_provisioning_service>` for the `provisioning service <nRF Cloud Provisioning Service_>`_, you can provision and onboard your device in one of two ways:
-
-* Using auto-onboarding, the easiest method.
+If you are not using nRF91x1, proceed to :ref:`nrf_cloud_multi_service_standard_onboarding`.
 
-   The nRF Cloud Provisioning Service auto-onboarding is compatible with CoAP, REST and MQTT connectivity with nRF Cloud.
+.. important::
+   Your project needs to be build with :ref:`Provisioning service support <nrf_cloud_multi_service_building_provisioning_service>` otherwise the next steps will not work. See `Provisioning Service <nRF Cloud Provisioning Service_>`_ for more detail information.
```

Comment:
```suggestion
   Your project needs to be built with :ref:`Provisioning service support <nrf_cloud_multi_service_building_provisioning_service>` otherwise the next steps will not work. See `Provisioning Service <nRF Cloud Provisioning Service_>`_ for more detail information.
```